.github/workflows/freezeyt_blog.yaml

```diff
@@ -22,10 +22,10 @@ jobs:
     steps:
       # Checks out a copy of your repository on the ubuntu-latest machine
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
```

Comment:
I recommend using the new caching feature: https://github.com/actions/setup-python/releases/tag/v2.3.0